app/components/billing/process.js

```diff
@@ -65,6 +65,9 @@ export default Component.extend({
         const currentStep = this.steps[nextIndex];
         this.set('currentStep', currentStep);
         this.persistBillingData(currentStep);
+        if (this.moveToNextBillingStep) {
+          this.moveToNextBillingStep();
+        }
```

Comment:
As long as the `moveToNextBillingStep` seems to be a handler that's assigned from the outside, I'd suggest to define it as a no-op method on the component interface:
```js
moveToNextBillingStep() {}
...
someAction() {
  ...
  this.moveToNextBillingStep()
  ...
}
```
In this case, if it's assigned from the outside, it'll overwrite the no-op, otherwise, it won't brake the app, even without guards. So it's a bit safer. And as a bonus, it makes the component public interface a bit more explicit.